UI/Pages/TuitionPartner.cshtml

```diff
@@ -249,6 +249,39 @@
                 </govuk-summary-list-row-value>
             </govuk-summary-list-row>
         }
+        @if (!string.IsNullOrWhiteSpace(Model.Data.ImportId))
+        {
+            <govuk-summary-list-row data-testid="tp-meta-data">
+                <govuk-summary-list-row-key>
+                    Import Id
+                </govuk-summary-list-row-key>
+                <govuk-summary-list-row-value>
+                    @(Model.Data.ImportId)
+                </govuk-summary-list-row-value>
+            </govuk-summary-list-row>
+        }
+        @if (Model.Data.TPLastUpdatedData.HasValue)
+        {
+            <govuk-summary-list-row>
+                <govuk-summary-list-row-key>
+                    Data last updated by @Model.Data.Name
+                </govuk-summary-list-row-key>
+                <govuk-summary-list-row-value>
+                    @(Model.Data.TPLastUpdatedData)
+                </govuk-summary-list-row-value>
+            </govuk-summary-list-row>
+        }
+        @if (Model.Data.ImportProcessLastUpdatedData.HasValue)
```

Comment:
What happens if we have DateTime default value `i.e: 01/01/0001 00:00:00`? Do we show it?

Reply:
Yep, that's correct.  I don't think this is a big problem, since it's only to be used (via the query string) via DfE staff?

Reply:
Yeah, I see your point. Please mention that on the ticket as well for the others.